server/supernova/CMakeLists.txt

```diff
@@ -104,31 +104,57 @@ else()
   endif()
 endif()
 
-if(APPLE OR MSYS)
+if(AUDIOAPI STREQUAL "default")
+	if(APPLE)
+		set(AUDIOAPI portaudio)
+	elseif(WIN32)
+		set(AUDIOAPI portaudio)
+	else()
+		set(AUDIOAPI jack)
+	endif(APPLE)
+endif()
+
+if(NOT AUDIOAPI MATCHES "^(jack|coreaudio|portaudio)$")
+	message(FATAL_ERROR "Unrecognised audio API: ${AUDIOAPI}")
+endif()
+
+if(AUDIOAPI STREQUAL "coreaudio")
+	set(AUDIOAPI portaudio)
+  message(STATUS "Supernova: coreaudio API not implemented, using ${AUDIOAPI} instead")
+endif()
+
+if(AUDIOAPI STREQUAL jack)
+    find_package(Jack)
+    if(NOT JACK_FOUND)
+        message(FATAL_ERROR "Jack selected as audio API, but development files not found")
+    endif()
```

Comment:
`find_package(Jack REQUIRED)` will accomplish the same thing more effectively. you could also update the portaudio check below the same way

https://cmake.org/cmake/help/v3.5/command/find_package.html